lib/streamlit/elements/exception_proto.py

```diff
@@ -177,20 +175,11 @@ def _get_stack_trace(
     """
     # Get and extract the traceback for the exception.
     extracted_traceback = None  # type: Optional[traceback.StackSummary]
```

Comment:
`mypy` failed in the Python 3.8 test suite (the only place it runs) because `traceback.extract_tb()` is annotated as returning a `traceback.StackSummary`, without the possibility of `None`, and because of that `if extracted_traceback is None:` around line 185 is known to evaluate as `False` and its body is unreachable.

You should remove line 177 where we initialize `extracted_traceback` (needed for the removed Python 2 case), and then change the if/else around line 185 to only retain the body of the `else`, since the `if` body will never execute. That'll fix that `mypy` failure.